front-end/src/pages/AuthorPage.tsx

```diff
@@ -41,16 +43,18 @@ export default function AuthorPage(props: any) {
       console.log("ERROR GETTING AUTHOR INFO");
       setResponseMessage(500);
     })
+
     if (props.loggedInUser) {
-      // get whether user is follower of author
-      axios.get(authorUrl + "/followers/" + props.loggedInUser.authorId).then(res => {
-        setIsFollower(true);
-      }).catch(err => {
-        // 404 is not a follower
-        setIsFollower(false);
-      })
+      // get whether user is follower of author IF not looking at our own profile
+      if (!props.location.pathname.includes(props.loggedInUser.authorId)) {
```

Comment:
In the future we might want to avoid using `props.location.pathname`. React router has a way for us to get the exact parameters we want as props. 

Here is an example of how to do it (note the `MatchParams` interface): https://github.com/CMPUT404W21T02/group-project/blob/main/front-end/src/pages/PostDetailPage.tsx

https://stackoverflow.com/questions/48138111/what-typescript-type-should-i-use-to-reference-the-match-object-in-my-props

Reply:
added a TODO comment. I think we can fix this when we get to work on the axios wrapper, since it would involve us having to change paths/URLs at that point